pysmt/optimization/goal.py

```diff
@@ -108,12 +111,13 @@ class MinimizationGoal(Goal):
     Attention: some Optimizer may not support this goal
     """
 
-    def __init__(self, formula):
+    def __init__(self, formula, sign = True):
```

Comment:
```suggestion
    def __init__(self, formula, signed=False):
```
sign = segno
signed = con-segno

signed suona meglio.

Moreover, I think that by default we shall be unsigned  (just like z3 and optimathsat)